src/parameter_declarations.ts

```diff
@@ -325,3 +327,14 @@ Parameter.declare({
 		'(OPTIONAL) Derives a drive key for the given drive ID without verifying its correctness against the drive on chain.',
 	type: 'boolean'
 });
+
+Parameter.declare({
+	name: GlobParameter,
+	aliases: ['-g', '--glob'],
+	description: `(BETA) a glob pattern to select files to be uploaded
+\t\t\t\t\t\t\t• Can NOT be used in conjunction with --local-files-path
+\t\t\t\t\t\t\t• Can NOT be used in conjunction with --local-files
+\t\t\t\t\t\t\t• Can lead to false positives or upload errors when glob string is not surrounded by quotation marks`,
+`,
+	forbiddenConjunctionParameters: [LocalFilesParameter, LocalFilePathParameter]
+});
```

Comment:
```suggestion

Parameter.declare({
	name: GlobParameter,
	aliases: ['-g', '--glob'],
	description: `(BETA) a glob pattern to select files to be uploaded
\t\t\t\t\t\t\t• Can NOT be used in conjunction with --local-files-path
\t\t\t\t\t\t\t• Can NOT be used in conjunction with --local-files
\t\t\t\t\t\t\t• Can lead to false positives or upload errors when glob string is not surrounded by quotation marks`,
	forbiddenConjunctionParameters: [LocalFilesParameter, LocalFilePathParameter]
});
```

Reply:
It was resolved

Reply:
@matibat It was a typo from the latest commit fixing the parameter description. I fixed it, can you try again?
Thank you!